docs/source/using/managing-minishift.adoc

```diff
@@ -275,3 +275,18 @@ Use the `openshift-version` option to request a specific version of OpenShift. Y
 
 The Minishift VM is exposed to the host system with a host-only IP address that
 can be obtained with the `minishift ip` command.
+
+[[connecting-with-ssh]]
+== Connecting to the Minishift VM with SSH
+
+You can use the xref:../command-ref/minishift_ssh{outfilesuffix}[`minishift ssh`] command to interact with the Minishift VM.
```

Comment:
This link does not work. Please verify links as part of the pull request. 

When using xrefs, we should not use {outfilesuffix} and we also need to specify an anchor. So a working example for this link is: xref:../command-ref/minishift_ssh.adoc#minishift-ssh[`minishift ssh`]